__tests__/components/Calculator/MaxScore.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import MaxScore from '../../../src/components/Calculator/MaxScore'
+
+const setup = (setupProps = {}) => {
+  const defaultProps = {
+    maxPrimaryScore: 182,
+    maxLingualScore: 192
+  }
+  const props = { ...defaultProps, ...setupProps }
+  const wrapper = shallow(<MaxScore {...props} />)
+
+  return {
+    props,
+    wrapper
+  }
+}
+
+describe('MaxScore component', () => {
+  test('should render', () => {
+    const { wrapper } = setup()
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should render total maxScore', () => {
+    const { wrapper } = setup()
+
+    expect(wrapper.find('b').text()).toEqual(` ${182 * 4 + 192} `)
```

Comment:
This `something * 4 + somethingElse` appears multiple times in 4 different files. It's some kind of max score formula AFAIK, but not obvious immediately. But even if you remember what this is, are you sure you want to hunt its usages in the application if the formula changes in the future?

Reply:
When formula changes I have to rebuild whole app. Isn't it normal to rebuild corresponding test files then too?

Reply:
If implementation/behaviour changes (new feature added/bug fixed) then other specs need to be updated as well.

Regarding the formula, if you have extracted business logic properly, then changes of the formula or other core concepts will require changes in only certain files, not the whole app.